iotile_ext_cloud/iotile/cloud/cloud.py

```diff
@@ -43,12 +43,15 @@ class IOTileCloud:
 
     def __init__(self, domain=None, username=None, **kwargs):
         reg = ComponentRegistry()
-        conf = ConfigManager()
+        self._conf = ConfigManager()
 
         if domain is None:
-            domain = conf.get('cloud:server')
+            domain = self._conf.get('cloud:server')
```

Comment:
I would recommend capturing the value of the config here and saving as a private variable `_verify`.  Since the impact of the config variable only really matters at the moment this constructor runs (since it's passed down and cached inside the created `Api` object, we shouldn't dynamically refetch the config variable when someone asks us if we're verifying the server's cert because it doesn't matter what the current value of the config object is, it only matters what the value was when the `IOTileCloud` object was created and it was cached inside `self.api`

Reply:
Makes sense, thanks. Updated - dc76d53676